menu/script.js

```diff
@@ -0,0 +1,35 @@
+const menuButton = document.querySelector("#menu-button");
+const menuContent = document.querySelector("#menu-content"); // takes two arguments every time, first one is event type as a string
+menuButton.addEventListener("click", () => {
+  event.preventDefault();
```

Comment:
If you want to target the 'event' you must explicitly name it as a parameter in line 3. So line 3 should look like: 

```menuButton.addEventListener("click", (event) => {```

Button default behavior doesn't need to be prevented here anyway.